contents/stats_distributions.md

```diff
@@ -0,0 +1,62 @@
+## Distributions {#sec:stats_dist}
+
+Before defining distributions, we must talk about **probability**.
+
+**The probability of an event is a real number ($\in \mathbb{R}$) between 0 and 1, where, roughly, 0 indicates the impossibility of the event and 1 indicates the certainty of the event**.
```

Comment:
R is the set of real numbers, but then what is a set? Probably a reference to R?

Reply:
This section is not ready. I've just dumped some stuff from https://github.com/storopoli/Bayesian-Julia to polish afterwards.